src/components/themed/TransactionListRow.tsx

```diff
@@ -116,13 +118,22 @@ export function TransactionListRow(props: Props) {
     })
   })
 
+  const handleLongPress = useHandler(() => {
+    const url = sprintf(currencyInfo.transactionExplorer, transaction.txid)
+    const shareOptions = {
+      url
+    }
+    Share.open(shareOptions).catch(e => console.log(e))
```

Comment:
Use `openBrowserUri` instead. This opens in an embedded SafariWebView on iOS

Reply:
looks like `openBrowserUri` opens non-embedded safari tab

https://user-images.githubusercontent.com/4023066/194687864-59112f9b-b036-4b3d-a985-91265c25aae0.mp4